controllers/routing/routing_fixtures_test.go

```diff
@@ -0,0 +1,85 @@
+package routing_test
+
+import (
+	"context"
+	"errors"
+
+	. "github.com/onsi/gomega"
+	"github.com/opendatahub-io/odh-platform/pkg/cluster"
+	"github.com/opendatahub-io/odh-platform/pkg/metadata"
+	"github.com/opendatahub-io/odh-platform/test"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+func getClusterDomain(ctx context.Context) string {
+	var domain string
+
+	Eventually(func(ctx context.Context) error {
+		var err error
+		domain, err = cluster.GetDomain(ctx, envTest.Client)
+
+		return err
+	}).WithContext(ctx).
+		WithTimeout(test.DefaultTimeout).
+		WithPolling(test.DefaultPolling).
+		Should(Succeed())
+
+	return domain
+}
+
+func exportCustomResource(ctx context.Context, exportedComponent *unstructured.Unstructured, mode string) {
+	// routing.opendatahub.io/export-mode: "public;external"
+	exposeExternally := metadata.WithAnnotations(metadata.Annotations.RoutingExportMode, mode)
+	_, errExportCR := controllerutil.CreateOrUpdate(
+		ctx, envTest.Client,
+		exportedComponent,
+		func() error {
+			return metadata.ApplyMetaOptions(exportedComponent, exposeExternally)
+		})
+	Expect(errExportCR).ToNot(HaveOccurred())
+}
+
+func addRoutingRequirementsToSvc(ctx context.Context, exportedSvc *corev1.Service, owningComponent *unstructured.Unstructured) {
```

Comment:
Maybe instead of comments in the func's body move them as godoc for the func and describe what it is intended to do?

Reply:
[63de885](https://github.com/maistra/odh-platform/pull/39/commits/63de88553f5a950c0ab9fbcb6273843c056546c4)